### PR TITLE
update from master with some tRO fix

### PR DESCRIPTION
**Pull request by mrpond**
new packetrecv

### CODE REVIEW

**Comment by Bilmola** [comment]
Not move in map . . and disconnect.

**Comment by arphongbas** [comment]
same me cant move and disconnect T_T

**Comment by mrpond** [comment]
it not for noob, you still need todo EAC bypass.
this is only openkore packet

**Comment by arphongbas** [comment]
how to EAC bypass i dont know

**Comment by ksuebkaew** [comment]
> how to EAC bypass i dont know

that is the things we need to figure it out...

File: src/Network/Receive.pm [hunk 1]
```diff
@@ -1088,13 +1100,22 @@ sub account_server_info {
...
 	@servers = @{$args->{servers}};
-
-	my $msg = center(T(" Servers "), 53, '-') ."\n" .
-			T("#   Name                  Users  IP              Port\n");
-	for (my $num = 0; $num < @servers; $num++) {
-		$msg .= swrite(
-			"@<< @<<<<<<<<<<<<<<<<<<<< @<<<<< @<<<<<<<<<<<<<< @<<<<<",
-			[$num, $servers[$num]{name}, $servers[$num]{users}, $servers[$num]{ip}, $servers[$num]{port}]);
+	my $msg = center(T(" Servers "), 53, '-') ."\n";
+	if ($masterServer->{serverType} eq "tRO") {
```
Comment (sctnightcore) [inline src/Network/Receive.pm:1104]: I think dont need add new one ? .

`$servers[$num]{sid} ? $servers[$num]{sid} : '';
`

**Comment by minimax1** [comment]
how to EAC bypass i want to know bro

**Comment by Nipodemos** [comment]
no one knows man, that's why very few people are playing with bots
rumors say that a guy has the bypass and sells it, but i don't know who, where, nor how to contact

File: src/Network/Receive.pm (hunk 1 above)
Comment (mrpond) [reply src/Network/Receive.pm:1104]: Only tRO sent server select sent SID but not user count.
we need to remove user column in case of tRO.

**Comment by TanasanJJ7** [comment]
I have rented a bot from the ukore-team already have this file. If it is useful
[https://upload.i4th.in.th/getlinks.php?s=b7ea4c328481c152a457cd897e54d47a](url)

**Comment by onstop** [comment]
> I have rented a bot from the ukore-team already have this file. If it is useful
> https://mega.nz/file/BWRjwK5S#cPVadah4gTIb1zYKOsMtUgZ0cnSdCADBMFmVEIKEHu0
> 
> May be a way to EAC Bypass
> 
> [](https://www.img.in.th/image/UL6rTj)

**Comment by onstop** [comment]
> I have rented a bot from the ukore-team already have this file. If it is useful
> https://mega.nz/file/BWRjwK5S#cPVadah4gTIb1zYKOsMtUgZ0cnSdCADBMFmVEIKEHu0
> 
> May be a way to EAC Bypass
> 
> []

I don't see those files

**Comment by jaturapat1** [comment]
เข้าได้แต่เดินไม่ได้ ทำไงคับ

**Pull request closed without merging**